crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2826,17 +2831,18 @@ impl Room {
         Ok(())
     }
 
-    /// Get the beacon information event in the room for the current user.
+    /// Get the beacon information event in the room for the `user_id`.
     ///
     /// # Errors
     ///
     /// Returns an error if the event is redacted, stripped, not found or could
     /// not be deserialized.
     async fn get_user_beacon_info(
         &self,
+        user_id: &UserId,
```

Comment:
Why are you adding the `user_id` argument here? You're using the method the same way by passing `Self::own_user_id()` anyway. Moreover, I'm not sure it's possible to get the beacon info for **another** user within a client.

Reply:
See https://github.com/matrix-org/matrix-rust-sdk/pull/4025#discussion_r1773745174 above